### DISCUSSION TITLE
Why can't I login to cloud.signalk.org

### DISCUSSION THREAD

**Discussion by sbender9** [sbender9/signalk-cloud#10] [FAQ]
You do not need to login to the cloud server. You can see your data and use the web apps, etc. without logging in.

**Comment by bobeasterday** [comment]
I must be missing something. clicking on my ws......@gmail.com link in the dashboard takes me to a login page. Entering my gmail credentials results in a failed login.

**Reply by sbender9** [reply]
As it says above, you do not need to login.

**Reply by AlanJHancock** [reply]
I had the same problem thinking my login was not working. Yes, I can see my data but only if I am selecting the correct context entry on the Data Browser and that could be one of many. Surely clicking the ws.myname@gmail_com link should take me to my data and not a login screen  The Web Apps do not seem to work either some request a login and others do not work like KIP which gives the error 'Connection To Server Lost'. I am a newbie to SignalK but I think something is broken or am I missing something?

**Reply by ReedStr** [reply]
For the next person, like myself, who is confused by this issue, you cannot see your data by clicking on your boat name in the dashboard.  You must go the Data Browser and find your boat on the context menu usingnidentifiers like your mmsi number.  Similarly, the aps like wilhelmsk don’t need a login but if you are having troup\blemconnecting after using the signalk data source try uninstalling and reinstalling.  It does work.
